modules/oc-mirror-updating-cluster-manifests.adoc

```diff
@@ -29,6 +27,13 @@ The `ImageContentSourcePolicy` resource associates the mirror registry with the
 $ oc apply -f ./oc-mirror-workspace/results-1639608409/
 ----
 
+. Apply the release image signatures to the cluster:
```

Comment:
```suggestion
. Apply the release image signatures to the cluster by running the following command:
```
While you are here, it might be worth updating this procedure to adhere to the guidelines here [1].

[1] https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output